cypress.config.js

```diff
@@ -1,8 +1,29 @@
-const { defineConfig } = require('cypress')
+const { cypress } = require('cypress');
```

Comment:
Replace the incorrect import statement `const { cypress } = require('cypress');` with `const cypress = require('cypress');`.